README.md

```diff
@@ -387,6 +387,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :books: [This is CS50x](https://cs50.harvard.edu/x/2021) - David J. Malan
 - :books: [IIT Bombay Foundation of Data Structures (CS213.1x)](https://courses.edx.org/courses/course-v1:IITBombayX+CS213.1x+1T2017/course/)
 - :books: [Intro to Data Structures and Algorithms](https://www.udacity.com/course/data-structures-and-algorithms-in-python--ud513) - Brynn Claypoole, Horatio Thomas
+- :📚: [Practise Excercise Level 1 DSA Pepcoding](https://pepcoding.com/resources/online-java-foundation) - Sumeet Malik
```

Comment:
Please place this under Java. Also, you should write "books" rather than the emoji itself.

Just copy this `:books:` instead